### PR TITLE
Adding new awesome speaker from Wroclaw

### PR DESCRIPTION
**Pull request by konradck**
I've added <person(s)> and I've let them know that they're on the list here:

<url to tweet, ex. https://twitter.com/karlhorky/status/927550556570357760>

### CODE REVIEW

**Comment by SaraVieira** [comment]
Hello !

Can you please let Mike know you are adding him here ?

Thank you

File: README.md [hunk 1]
```diff
@@ -619,6 +619,12 @@ Mike Grabowski
 Topics: React Native, JavaScript, Tooling  
 https://twitter.com/grabbou
 
+<img src="https://pbs.twimg.com/profile_images/813866423214870528/ct8REef9_400x400.jpg" height="70px" width="auto" align="left" alt="" />
+
+Mike Pierzchala
```
Comment (thymikee) [inline README.md]: nit: `Michal Pierzchala` (or `Michał Pierzchała` if you fancy non-latin chars, I'm OK with both)

I'm also available in Krakow :D but nevermind

**Review by thymikee — APPROVED** [review]
Wow, so nice to be here! All good but a minor nit I commented on :)

**Comment by SaraVieira** [comment]
@thymikee I changed the name :)

Does it look okay now ?

**Comment by thymikee** [comment]
Awesome 👍

**Comment by SaraVieira** [comment]
Merged ! Thank you both ❤️

**Pull request merged**